systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -192,16 +190,18 @@ void testKafkaPodImagePullBackOff(ExtensionContext extensionContext) {
 
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaPersistent(clusterName, 3, 3).build());
 
+        String kafkaImage = KafkaResource.kafkaClient().inNamespace(NAMESPACE).withName(clusterName).get().getSpec().getKafka().getImage();
```

Comment:
Who and where puts the image into the Kafka spec?

Reply:
You can set it there. But by default it is not - for example in the examples. So you would need some other part of the ST to set it there?